assets/launcher/lang/sklmessages_pl_PL.properties

```diff
@@ -40,7 +40,7 @@ game.output.type.text=tekstowy
 
 game.conflict.title=Ostrzeżenie o zduplikowanej instancji
 game.conflict.header=Posiadasz już uruchomioną instancję Minecrafta.
-game.conflict.text=Jeśli uruchomisz kolejną z tego samego folderu, może dojść do konfliktu i uszkodzenia twoich zapisów gry. \nMoże to stworzyć wiele problemów, w grze jednoosobowej lub w inny sposób. Nie będziemy ponosić odpowiedzialności za cokolwiek, co pójdzie nie tak. \nCzy mimo to chcesz uruchomić kolejną instancję Minecrafta? \nMoże udać ci się zapobiec problemom, jeżeli uruchomisz grę w innym folderze (patrz: przycisk "Edytuj profil").
+game.conflict.text=Jeśli uruchomisz kolejną z tego samego katalogu, może dojść do konfliktu i uszkodzenia Twoich zapisów gry. \nMoże to stworzyć wiele problemów, w grze jednoosobowej lub w inny sposób. Nie będziemy ponosić odpowiedzialności za cokolwiek, co pójdzie nie tak. \nCzy mimo to chcesz uruchomić kolejną instancję Minecrafta? \nMoże uda Ci się zapobiec problemom, jeżeli uruchomisz grę w innym katalogu (patrz: przycisk "Edytuj profil").
```

Comment:
Ja bym zbytnio nie przesadzał z dużymi literami i grzecznością, to w końcu launcher, a nie list